src/client/post/EditPost/EditPost.js

```diff
@@ -137,21 +141,21 @@ class EditPost extends Component {
       const serverObject = await getObject(forecastObjectId);
       forecastObject = getClientWObj(serverObject, this.props.locale);
     }
-
-    let linkedList = [forecastObject, ...objEntities.map(entity => entity.data.object)];
-    if (!isRemovedCryptoObj) linkedList = [cryptoObject, ...linkedList];
-
-    return compact(uniqBy(linkedList, 'id'));
+    return compact(uniqBy([forecastObject, ...objEntities.map(entity => entity.data.object)], 'id'));
   };
 
   async handleChangeContent(rawContent) {
+    const { defaultObjects } = this.state;
     const nextState = { content: toMarkdown(rawContent) };
     this.getLinkedObjects(rawContent).then(linkedObjects => {
       const isLinkedObjectsChanged = !isEqual(this.state.linkedObjects, linkedObjects);
+      const linkedList = isLinkedObjectsChanged ? linkedObjects : this.state.linkedObjects;
+      const postObjects = !isEmpty(defaultObjects) ? [...defaultObjects, ...linkedList] : linkedList;
       if (isLinkedObjectsChanged) {
-        const objPercentage = setObjPercents(linkedObjects, this.state.objPercentage);
         nextState.linkedObjects = linkedObjects;
-        nextState.objPercentage = objPercentage;
+        nextState.objPercentage = setObjPercents(postObjects, this.state.objPercentage);
+      } else {
+        nextState.objPercentage = setObjPercents(postObjects, this.state.objPercentage);
```

Comment:
why do you use 'else' branch? Can we omit it?

Reply:
++